gitlab/integration_test.go

```diff
@@ -627,6 +627,69 @@ var _ = Describe("GitLab Provider", func() {
 		}
 	})
 
+	It("should create, delete and reconcile deploy tokens", func() {
+		testDeployTokenName := "test-deploy-token"
+		repoRef := newOrgRepoRef(testOrgName, testSharedOrgRepoName)
```

Comment:
This makes the spec dependent on other specs being run (particularly the one that creates the shared repo). We shouldn't do that.

Reply:
What's the alternative? I've pretty much copied that from the [deploy key spec](https://github.com/timofurrer/go-git-providers/blob/3dc0257a917a964d4e2730cd0df2102bfe178a51/gitlab/integration_test.go#L559)

Reply:
I know, the existing tests aren't the best examples. 😞 Each test should actually create its own repository. Let me see if I can submit a commit that makes this possible w/o too much overhead per-test.